packages/neos-ui-editors/src/Library/LinkInput.js

```diff
@@ -149,6 +149,10 @@ export default class LinkInput extends PureComponent {
     }
 
     handleSearchTermChange = searchTerm => {
+        // trim leading and trailing whitespace as it can cause issues
+        // with the further processing
+        searchTerm.trim()
```

Comment:
Could there be a case in which `searchTerm` is null or undefined?

Reply:
Right, that should be checked!

Reply:
Normally, it should be always a string, but it is better to prevent a potential error